### PR TITLE
Fix loader

### PR DESCRIPTION
**Pull request by NatalyMac**
Add reader-critics loader after user clicked on the Send review (feedback) button

### CODE REVIEW

File: src/front/feedback/FeedbackContainer.tsx [hunk 1]
```diff
@@ -119,19 +122,26 @@ extends React.Component <any, FeedbackContainerState> {
...
+		const { isFeedbackReady, isSending } = this.state;
+		const feedbackForm = this.renderFeedbackForm();
+		const confirmationPage = this.renderConfirmationPage();
+		return (
+			<div>
+			{ isFeedbackReady && !isSending && confirmationPage }
+			{ !isFeedbackReady && !isSending && feedbackForm }
+			{ isSending && !isFeedbackReady &&  <Spinner/> }
+			</div>
+		);
```
Comment (EugeneSapozhkov) [inline src/front/feedback/FeedbackContainer.tsx]: Probably better use instant function not variable.
And maybe better optimize conditions, if you are sending something, you don't need look at feedback variable, and something similar happened when feedback was returned )

{ !isFeedbackReady && !isSending &&  this.renderFeedbackForm(); }
{ isFeedbackReady && this.renderConfirmationPage(); }
{ isSending && <Spinner/> }

File: src/front/feedback/FeedbackContainer.tsx [hunk 2]
```diff
@@ -105,9 +110,7 @@ extends React.Component <any, FeedbackContainerState> {
 			alert(<FormattedMessage id="fb.errors.emptyErr"/>);
 			return;
 		}
-
 		console.log('sending feedback');
```
Comment (EugeneSapozhkov) [inline src/front/feedback/FeedbackContainer.tsx]: Forgot to remove console.log?

Comment (EugeneSapozhkov) [reply src/front/feedback/FeedbackContainer.tsx]: Need to clean code little bit ) and remove debug comments

Comment (NatalyMac) [reply src/front/feedback/FeedbackContainer.tsx]: legacy, removed

**Pull request merged**